test/lib/graphql/schema_comparator_test.rb

```diff
@@ -28,6 +28,7 @@ def test_compare_handles_idls
     assert_equal [
       "Field `Query.a` changed type from `String!` to `Int`",
       "Field `b` was added to object type `Query`",
+      "Type `Int` was added",
```

Comment:
🤔  I guess the gem only adds built-in scalars if they're _used_ now, which it didn't before? 🤷

Reply:
Might be from https://github.com/rmosolgo/graphql-ruby/pull/2722